pkg/systemd/overview-cards/lastLogin.jsx

```diff
@@ -125,6 +129,15 @@ const LastLogin = () => {
                         {lastLoginText}
                     </FlexItem>
                     }
+                    {name &&
+                    <FlexItem>
+                        <Button variant="link" isInline
```

Comment:
Generally we should use `<a>` component when we get redirected to other pages, also inside cockpit. That said component='a' here with an href. 

However it's wrong all over cockpit, one more won't make a difference. We can fix it in a big batch in the future.